util/src/main/java/tech/pegasys/artemis/util/SSZTypes/SSZVector.java

```diff
@@ -39,6 +42,17 @@ public SSZVector(List<T> list, Class<T> classInfo) {
     this.classInfo = classInfo;
   }
 
+  @JsonCreator
+  @SuppressWarnings("unchecked")
+  public SSZVector(List<T> list) {
+    super(list);
+    maxSize = list.size();
+    if (maxSize < 1) {
+      throw new UnsupportedOperationException("SSZVector must have specified size");
```

Comment:
should it say "must have specified size > 0"

Reply:
yep it can do - this is just the operation exception from above